### PR TITLE
ug: lmp customization: add steps to enable ntp

### PR DESCRIPTION
**Pull request by vanmaegima**
Common customer request.

## Readiness

* [ ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 162fd42b71a2a1494d6189d2f98d8f466ce92373 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2051/docs/artifacts/html/index.html

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Comment by ricardosalveti** [comment]
Along with this switch, it might be good to document how to customize the standard ntp server list when using timesyncd (https://github.com/foundriesio/meta-lmp/blob/main/meta-lmp-base/recipes-core/systemd/systemd_%25.bbappend#L60).

**Comment by vanmaegima** [comment]
I'm not really sure why the link-check is failing, https://wiki.st.com/stm32mpu/wiki/STM32CubeProgrammer#STM32CubeProgrammer_installation looks valid to me.

File: source/user-guide/lmp-customization/lmp-customization.rst [hunk 1]
```diff
@@ -323,3 +323,30 @@ Adding LmP Users
 
 After these changes, the files ``/usr/lib/passwd`` and ``/usr/lib/group`` should include the configuration for the new user.
 
+LmP time servers
```
Comment (kprosise) [inline source/user-guide/lmp-customization/lmp-customization.rst:326]:
```suggestion
LmP Time Servers
```

File: source/user-guide/lmp-customization/lmp-customization.rst [hunk 2]
```diff
@@ -323,3 +323,30 @@ Adding LmP Users
...
+Then, enable ``ntp`` in ``meta-subscriber-overrides/recipes-samples/images/lmp-factory-image.bb`` by appending the ``CORE_IMAGE_BASE_INSTALL`` variable:
+
+.. code-block:: none
+
+    CORE_IMAGE_BASE_INSTALL += " \
+        ntp \
+    "
+
+.. note::
+    If ``systemd-timesyncd`` is used, the default ``ntp`` server list is set `here <https://github.com/foundriesio/meta-lmp/blob/main/meta-lmp-base/recipes-core/systemd/systemd_%25.bbappend>`_::
```
Comment (kprosise) [inline source/user-guide/lmp-customization/lmp-customization.rst:346]:
```suggestion
    If ``systemd-timesyncd`` is used, the default ``ntp`` server list is set `in this recipe <https://github.com/foundriesio/meta-lmp/blob/main/meta-lmp-base/recipes-core/systemd/systemd_%25.bbappend>`_::
```

**Review by kprosise — APPROVED** [review]
Two suggestions based on Linter findings. Otherwise it LGTM

**Comment by doanac** [comment]
Docs for 484e1c8e4432c68e723409f696abd4afcb360490 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2057/docs/artifacts/html/index.html

**Pull request merged**